llvm/lib/SYCLLowerIR/LowerWGScope.cpp

```diff
@@ -425,6 +437,17 @@ static void copyBetweenPrivateAndShadow(Value *L, GlobalVariable *Shadow,
   }
 }
 
+// Skip allocas, addrspacecasts associated with allocas and debug insts.
+Instruction *getFrontInstAfterSkippingAllocas(BasicBlock *BB) {
```

Comment:
Function name is a bit misleading, maybe `getFirstInstToProcess` or something? Anything is fine

Reply:
The existing codebase used 'front'. So, I tried to keep the trend. I will rethink. Thanks